server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailGet.scala

```diff
@@ -110,7 +110,7 @@ case class EmailGetRequest(accountId: AccountId,
           bodyValues = bodyValues),
         specificHeaders = properties.getOrElse(Properties.empty()).value
           .filter(property => property.startsWith(SPECIFIC_HEADER_PREFIX))
-          .map(property => (sanitizeHeader(property), asEmailHeaderValue(mime4JMessage.getHeader.getField(property.substring(SPECIFIC_HEADER_PREFIX.length)))))
+          .map(property => (property.value, asEmailHeaderValue(mime4JMessage.getHeader.getField(property.substring(SPECIFIC_HEADER_PREFIX.length)))))
```

Comment:
:-( we need still to better extract logic here.

Rule of thumb. No string manipulation in an out-of-bound context (that's a red line you've crossed here)